src/components/Counter.js

```diff
@@ -47,7 +46,7 @@ class Counter extends Component {
 // parts it needs from the state object.
 const mapStateToProps = (state) => {
     return {
-        count: state.count
+        count: state
```

Comment:
```suggestion
        count: state.count
```
Better to be specific here with count: state.count, because all we want from our state object is the count value. What if you had more than one property in your state object? Then you would be setting count: to equal all the properties in your state object, which wouldn't make much sense.